extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -110,7 +110,7 @@ export class DeployConfigPage extends DacFxConfigPage {
 
 		let newRadioButton = this.view.modelBuilder.radioButton()
 			.withProperties({
-				name: 'updateExisting',
+				name: 'newDatabase',
```

Comment:
for radio buttons, name is the group that the radio button is in, so upgradeRadioButton and newRadioButton should have the same name. Maybe the name should be changed to something like 'newOrUpgradeExisting' so it's less confusing.

Reply:
ouch we need something different to differentiate otherwise test wont work. Any other field I could use? may be label or something?

Reply:
Thanks for the save Kim! changed to label